### DISCUSSION TITLE
Use as python module

### DISCUSSION THREAD

**Discussion by meet-minimalist** [pranjal-joshi/Screeni-py#243] [General]
Hi,
First of all big thanks for creating something like this. I want to further extend this in my application by importing this module and using its functionality. I think currently it is developed as a streamlit app but I want to see this as a module to integrate with my code. Is it possible?

**Comment by pranjal-joshi** [comment]
Hi @meet-minimalist 

Yes! You can consume this while following certain good practices of the open source community.
1. Make sure to fork the Screenipy repo and then make changes on top of that instead of cloning/downloding source code and pushing it back as a new repo!
2. Avoid manipulating Author names in scripts (Feel free to add new but let the existing one be there for their credits)
3. Do not copy the entire codebase and just rename it something else from Screeni-py (Ensure to motivate the community by giving credits where it is due)
4. Feel free to raise a PR back to Screeni-py if you think your feature can help others (Optional)
